src/System.Runtime.Extensions/src/System/IO/Path.Unix.cs

```diff
@@ -213,5 +239,13 @@ private static unsafe void GetCryptoRandomBytes(byte* bytes, int byteCount)
                 throw new InvalidOperationException(SR.InvalidOperation_Cryptography);
             }
         }
+
+        // This P/Invoke is normally handled by the crypto shim, but currently calling
+        // the Apple-based crypto shim requires cross-compiling, and that is a very big
+        // hammer for calling a very stable system API. If this assembly ever gets split
+        // such that OSX and Unix are different compilations, this should move to standard
+        // interop patterns.
+        [DllImport("/usr/lib/libSystem.B.dylib")]
+        private static unsafe extern int CCRandomGenerateBytes(byte* bytes, int byteCount);
```

Comment:
I was keeping it out so that no one else used it (everyone else should be using the public API, really); but I felt dirty about that... so the question is enough to make me agree to move it.